### PR TITLE
Add coverage for content view functionality (BZ1416857)

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_update_name
..
----------------------------------------------------------------------
Ran 2 tests in 70.519s

OK
```
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_delete
..S....
----------------------------------------------------------------------
Ran 7 tests in 600.744s

OK (SKIP=1)
```
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_create_composite_by_name
.
----------------------------------------------------------------------
Ran 1 test in 117.070s

OK
```
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_remove_component_by_name
.
----------------------------------------------------------------------
Ran 1 test in 121.038s

OK
```
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_clone
....
----------------------------------------------------------------------
Ran 4 tests in 238.678s

OK
```

### CODE REVIEW

File: tests/foreman/cli/test_contentview.py [hunk 1]
```diff
@@ -773,6 +816,55 @@ def test_positive_create_composite(self):
...
+        new_cv = ContentView.info({u'id': new_cv['id']})
+        cvv = new_cv['versions'][0]
+        # Create CV
+        cv = make_content_view({
+            'composite': True,
+            'organization-id': self.org['id'],
+        })
+        # Associate version to composite CV
+        ContentView.add_version({
+            u'content-view-version-id': cvv['id'],
```
Comment (abalakh) [inline tests/foreman/cli/test_contentview.py]: So it's added by id, not by name as test name/desription says.

Comment (abalakh) [inline tests/foreman/cli/test_contentview.py]: Hm, test description has two meanings :) 'its name' could be applied to non-composite an composite CV. Anyway, I'd rather use names for both CVs in this test and ids for CVs in `by_id` test for better coverage.

File: tests/foreman/cli/test_contentview.py [hunk 2]
```diff
@@ -821,6 +913,49 @@ def test_positive_remove_version_by_id_from_composite(self):
...
+        # Create a composite CV
+        comp_cv = make_content_view({
+            'composite': True,
+            'organization-id': self.org['id'],
+            'component-ids': new_cv['versions'][0]['id']
+        })
+        self.assertEqual(len(comp_cv['components']), 1)
+        ContentView.remove_version({
+            u'content-view-version-id': new_cv['versions'][0]['id'],
+            u'id': comp_cv['id'],
```
Comment (abalakh) [inline tests/foreman/cli/test_contentview.py]: Neither CV name, nor component name are specified here. Please update the test accordingly.

Comment (oshtaier) [inline tests/foreman/cli/test_contentview.py]: content-view-version is a component name. I know that naming is a very confusing, but can't help with that
Read carefully https://bugzilla.redhat.com/show_bug.cgi?id=1320146 for any details needed.

But you are right, I needed to use name instead id. My mistake. Let's me fix it

**Comment by oshtaier** [comment]
Using name wherver I can
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_remove_component_by_name
.
----------------------------------------------------------------------
Ran 1 test in 127.566s

OK
```

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_remove_component_by_name
.
----------------------------------------------------------------------
Ran 1 test in 128.662s

OK
```

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/cli/test_contentview.py -m test_positive_create_composite_by_name
.
----------------------------------------------------------------------
Ran 1 test in 123.709s

OK
```

**Pull request merged**